usr/bin/sandbox-app-launcher

```diff
@@ -40,6 +67,40 @@ if [ "${app_name}" = "torbrowser" ]; then
   app_path="${app_homedir}/.tb/tor-browser/Browser/start-tor-browser"
 fi
 
+if ! [ -x "${app_path}" ] && ! [ "${app_name}" = "torbrowser" ]; then
+  echo "ERROR: Could not find '${app_name}' in \$PATH." >&2
+  exit 1
+fi
+
+if [ "${app_name}" = "shared" ]; then
+  echo "ERROR: The app name cannot be 'shared'." >&2
+  exit 1
+fi
+
+if [ -f "/etc/sandbox-app-launcher/${app_name}.conf" ]; then
+  . "/etc/sandbox-app-launcher/${app_name}.conf"
+fi
+
+## Optionally allow dynamic native code execution.
+##
+## This allows creating memory mappings that are both
+## writable and executable, allows transitioning a
+## writable memory mapping to executable and allows
+## executing programs from writable directories i.e.
+## violating W^X.
+##
+## This is generally a security issue since it allows an
+## attacker to execute new arbitrary code so preventing this
+## will force the attacker to use the already existing code
+## (ROP/JOP) which is far more limited.
+##
+## Although, some things require this such as JIT engines in
+## browsers so it must be optional.
+if [ "${allow_dynamic_native_code_exec}" = "yes" ]; then
+  seccomp_filter="${auto_dir}/seccomp-filter-wx.bpf"
+  wrapper_dir="${wrapper_dir_wx}"
```

Comment:
Setting `wrapper_dir="${wrapper_dir_wx}"` here does not influence contents of variable `wrapper_script`. It would stay as in above example.

> /var/cache/sandbox-app-launcher-autogenerated/wrappers/firefox

But should `wrapper_script` become

> /var/cache/sandbox-app-launcher-autogenerated/wrappers-wx/firefox

at that point?

Reply:
Good catch. Fixed, https://github.com/Whonix/sandbox-app-launcher/pull/46/commits/97939c6eb14d358581cb993b88f079fa4ede4ef7